index.html

```diff
@@ -0,0 +1,168 @@
+        </section>
+        <section class="speakers">
+            <h2>Our Keynote Speakers</h2>
+            <hr>
+            <div class="speakers-container">
+                <article class="speaker speaker-1">
+                    <div class="speaker-img-container">
+                        <img class="brain" src="graphics/brain-circuit.png" alt="">
+                        <img class="speaker-img" src="src" alt="">
+                    </div>
+                    <div class="speaker-content">
+                        <h3></h3>
+                        <div class="speaker-subheading"></div>
+                        <hr>
+                        <p></p>
+                    </div>
+                </article>
+                <article class="speaker speaker-2">
+                    <div class="speaker-img-container">
+                        <img class="brain" src="graphics/brain-circuit.png" alt="">
+                        <img class="speaker-img" src="src" alt="">
+                    </div>
+                    <div class="speaker-content">
+                        <h3></h3>
+                        <div class="speaker-subheading"></div>
+                        <hr>
+                        <p></p>   
+                    </div>
+                </article>
+                <article class="speaker speaker-3">
+                    <div class="speaker-img-container">
+                        <img class="brain" src="graphics/brain-circuit.png" alt="">
+                        <img class="speaker-img" src="src"  alt="">
+                    </div>
+                    <div class="speaker-content">
+                        <h3></h3>
+                        <div class="speaker-subheading"></div>
+                        <hr>
+                        <p></p>  
+                    </div>
+                </article>
+                <article class="speaker speaker-4">
+                    <div class="speaker-img-container">
+                        <img class="brain" src="graphics/brain-circuit.png" alt="">
+                        <img class="speaker-img" src="src" alt="">
+                    </div>
+                    <div class="speaker-content">
+                        <h3></h3>
+                        <div class="speaker-subheading"></div>
+                        <hr>
+                        <p></p> 
+                    </div>
+                </article>
+                <article class="speaker speaker-5">
+                    <div class="speaker-img-container">
+                        <img class="brain" src="graphics/brain-circuit.png" alt="">
+                        <img class="speaker-img" src="src" alt="">
+                    </div>
+                    <div class="speaker-content">
+                        <h3></h3>
+                        <div class="speaker-subheading"></div>
+                        <hr>
+                        <p></p>
+                    </div>
+                </article>
+            </div>
```

Comment:
- [ ] Hey, the speaker section should be added dynamically using javascript according to the project requirement. Please, kindly update these changes to meet the project requirement :pray: